includes/gateway/class-omise-payment-truemoney.php

```diff
@@ -24,6 +24,7 @@ public function __construct() {
 		$this->title                = $this->get_option( 'title' );
 		$this->description          = $this->get_option( 'description' );
 		$this->restricted_countries = array( 'TH' );
+		$this->source_type 			= 'truemoney';
```

Comment:
Indentation is not aligned.

Reply:
nice catch here fixed 🙏  https://github.com/omise/omise-woocommerce/pull/290/commits/c4557447a5d37397c7c9fa87246893b9d7d2dfb6